infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterDataV1.kt

```diff
@@ -96,13 +108,14 @@ sealed class FrameConverterRequestV1
  * @property locationTrackType User provided location track type filter, optional.
  * @property responseSettings User provided array of integers, which map to enum values, optional, has defaults.
  */
-data class CoordinateToTrackMeterRequestV1(
+data class CoordinateToTrackAddressRequestV1(
```

Comment:
Aiemmin käytössä ollut termi TrackMeter päätyi TrackAddress-termiin. Näitä oli vähän kumpaakin sekaisin ja päädyin ainakin toistaiseksi tähän addressiin